README.md

```diff
@@ -56,23 +56,23 @@ export interface ICard {
 ```
 
 # HTTP SDK
-The SilentEcho service can also be called directly via HTTP.
+The VirtualDevice service can also be called directly via HTTP.
 
 ## Pre-Requisites
-Get a SilentEcho Token:  
-[https://silentecho.bespoken.io/link_account?token=true](https://silentecho.bespoken.io/link_account?token=true)
+Get a VirtualDevice Token:
```

Comment:
They need to get this via the Dashboard URL now. This should say:
* Sign into the [Bespoken Dashboard](https://apps.bespoken.io/dashboard)
* Create a source
* Select the Validation tab and follow the instructions there